src/app/services/data.service.ts

```diff
@@ -123,6 +126,176 @@ export class DataService {
     }
     return Object.keys(result).sort().reverse();
   }
+  
+  public getRoutesLast30Days(): Route[] {    
+    var routesLast30Days = [];
+    const routes = this.getRoutes();
+    //TO DO: use object instead of array
```

Comment:
is this todo done?

for a comment leave a space after the two slashes

Reply:
no, and it was just sth Christopher kind of requested / suggested yesterday when looking at the code, since he did no like my nested for loop approach

However the method works fine as it is and i see at least no urgency to change it, do you?

I have added the spaces

j